pandas/core/generic.py

```diff
@@ -7974,8 +7995,85 @@ def resample(
         2000-01-02     22     140
         2000-01-03     32     150
         2000-01-04     36      90
+
+        If you want to adjust the start of the bins based on a fixed timestamp:
+
+        >>> start, end = "2000-10-01 23:30:00", "2000-10-02 00:30:00"
+        >>> rng = pd.date_range(start, end, freq="7min")
+        >>> ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
+        >>> ts
+        2000-10-01 23:30:00     0
+        2000-10-01 23:37:00     3
+        2000-10-01 23:44:00     6
+        2000-10-01 23:51:00     9
+        2000-10-01 23:58:00    12
+        2000-10-02 00:05:00    15
+        2000-10-02 00:12:00    18
+        2000-10-02 00:19:00    21
+        2000-10-02 00:26:00    24
+        Freq: 7T, dtype: int64
+
+        >>> ts.resample("17min").sum()
+        2000-10-01 23:14:00     0
+        2000-10-01 23:31:00     9
+        2000-10-01 23:48:00    21
+        2000-10-02 00:05:00    54
+        2000-10-02 00:22:00    24
+        Freq: 17T, dtype: int64
+
+        >>> ts.resample("17min", origin=pd.Timestamp("1970-01-01")).sum()
```

Comment:
hmm, how does this origin translate here? I actually find this confusing.

Reply:
@jreback Well origin is just a fixed point in time on which you decide to start the resampling. Using the epoch date was kind of handy in my mind:
* with the freq that divide or are a multiple of days: it works exactly in the same way
* with weird frequencies it's a fixed point in time kind of standard

But origin could have been any other date, we let the user decide what referential he wants to use.